### PR TITLE
merging adorable master to lmht master

### PR DESCRIPTION
**Pull request by ericanaughton**
:evergreen_tree: 

ready to go

A few PR's were merged to Adorable master, which were already approved. We just want another two thumbs before deploying this to the gh-page (merging to LMHT master).

Summary of changes:

*Updates README file
*Fixes typos
*Makes coding for symbols & Spanish accent marks consistent
*Restyles search tool (combobox/sticky-nav) to make it more of the main focus and easy to use
*Adds new Spanish translations

### CODE REVIEW

**Comment by ahaynes-ador** [comment]
:+1:

File: README.md [hunk 1]
```diff
@@ -1,79 +1,45 @@
...
-    <pre class="code-example">
-      <code>
-        <!-- Place your awesome code example here :D -->
-      </code>
-    </pre>
-  </article>
-```
+# LMHT - HTML: Lenguaje de Marca de Hipertexto - Hyper Text Markup Language
+
+Welcome to [LMHT].  This is an open source code repository to help Spanish speakers translate and understand HTML tags.  Originally started as a project by Team Open Brace in the [YWebCA] Summer of 2015 class, this project is now being developed further as an open source project by [Adorable] to expand it's possibilities.  With your help, we can expand this into an excellent learning tool for bilingual programmers.
```
Comment (m5rk) [inline README.md]: Typo: `it's possibilities` should be `its possibilities`.

File: README.md [hunk 2]
```diff
@@ -1,79 +1,45 @@
...
+## Introduction
+
+The [LMHT page] provides a list of current HTML tags.  Select the tag you are looking for or begin typing it in the search box.  Once selected, a separate window will pop up containing the english base of the tag, the spanish equivalent, a definition in both english and spanish as well as audio recordings of each of these items.  Furthermore, there are helpful hints on how to remember what the tag does as well as examples of the code being used.
+
+[LMHT page]:  http://lmht.github.io/
+
+
+## Known Issues
+
+* Attributes: Currently, in the beginning stages, the focus has been on creating the primary html tags.  Eventually, this will expand to cover attributes associated with each of these tags.
```
Comment (m5rk) [inline README.md]: Use a consistent style for this phrase: `HTML tags` vs `html tags`. You have both.

File: js/combobox.js [hunk 3]
```diff
@@ -8,6 +8,11 @@ $(function() {
          this.element.hide();
          this._createAutocomplete();
          this._createShowAllButton();
+// placeholder
```
Comment (m5rk) [inline js/combobox.js]: This comment doesn't add any value. Remove it.

File: js/combobox.js [hunk 4]
```diff
@@ -8,6 +8,11 @@ $(function() {
...
+         this.input.attr("placeholder", this.element.attr('placeholder'));
+         // clear the default selected option so that we see the placeholder text
+        //  this.input.value = '';
+        //  console.log(this.input)
```
Comment (m5rk) [inline js/combobox.js]: Remove commented out code--create an issue to track instead.

File: js/scripts.js [hunk 5]
```diff
@@ -5,6 +5,7 @@ var app = {
   init: function() {
     this.setUpMenu();
     this.setUpFirstView();
+    $('input').val(''); //todo: this code needs to be placed inside combobox.js somehow
```
Comment (m5rk) [inline js/scripts.js]: Is there an issue for this? If so, delete the comment?

**Comment by m5rk** [comment]
:+1: - None of the comments are merge blockers. Resolve them in this PR or later. I'd hate to hold up the merge for a few minor things.

**Comment by ericanaughton** [comment]
thanks @m5rk ! i'm making a new PR for these small fixes.

**Comment by ericanaughton** [comment]
Jim okayed this :+1:

**Pull request merged**